androidplot-core/src/main/java/com/androidplot/xy/PanZoom.java

```diff
@@ -95,7 +111,14 @@ public static PanZoom attach(XYPlot plot) {
     }
 
     public static PanZoom attach(XYPlot plot, Pan pan, Zoom zoom) {
-        PanZoom pz = new PanZoom(plot, pan, zoom);
+        PanZoom pz = new PanZoom(plot, pan, zoom,ZoomLimit.OUTER);
```

Comment:
I believe this method body can be changed to:

`return attach(plot, pan, zoom, ZoomLimit.OUTER);`